sdk/@launchdarkly/observability-android/lib/src/main/kotlin/com/launchdarkly/observability/interfaces/Metric.kt

```diff
@@ -0,0 +1,10 @@
+package com.launchdarkly.observability.interfaces
+
+import io.opentelemetry.api.common.Attributes
+
+data class Metric(
```

Comment:
Is this for encapsulation purposes, or does the otel SDK not have this type someplace?